lib/account/widgets/account_page_app_bar.dart

```diff
@@ -0,0 +1,212 @@
+
+  @override
+  void didUpdateWidget(covariant AccountPageAppBar oldWidget) {
+    super.didUpdateWidget(oldWidget);
+
+    if (showSaved != widget.showSaved && context.read<FeedBloc>().state.status == FeedStatus.success) {
+      setState(() => showSaved = widget.showSaved);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = AppLocalizations.of(context)!;
+    final state = context.read<ThunderBloc>().state;
+
+    return SliverAppBar(
+      pinned: true,
+      centerTitle: false,
+      titleSpacing: 0.0,
+      toolbarHeight: 70.0,
+      surfaceTintColor: state.hideTopBarOnScroll ? Colors.transparent : null,
+      title: AccountAppBarTitle(visible: widget.showAppBarTitle),
+      leading: IconButton(
+        onPressed: () {
+          HapticFeedback.mediumImpact();
+          showProfileModalSheet(context);
+        },
+        icon: Icon(Icons.people_alt_rounded, semanticLabel: l10n.profiles),
+        tooltip: l10n.profiles,
+      ),
+      actions: [
+        AccountAppBarUserActions(
+          showSaved: showSaved,
+          onToggleSaved: (showSaved) {
+            setState(() => this.showSaved = showSaved);
+            widget.onToggleSaved?.call(showSaved);
+          },
+        )
+      ],
+    );
+  }
+}
+
+/// The title of the app bar for the account page
+class AccountAppBarTitle extends StatelessWidget {
+  const AccountAppBarTitle({super.key, this.visible = true});
+
+  /// Whether to show the title. When the user scrolls down the title will be hidden
+  final bool visible;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final feedBloc = context.watch<FeedBloc>();
+    final person = feedBloc.state.fullPersonView?.personView.person;
+
+    return AnimatedOpacity(
+      duration: const Duration(milliseconds: 200),
+      opacity: visible ? 1.0 : 0.0,
+      child: ListTile(
+        title: Text(
+          person?.displayName ?? person?.name ?? '',
+          style: theme.textTheme.titleLarge,
+          maxLines: 1,
+          overflow: TextOverflow.ellipsis,
+        ),
+        subtitle: Text(
+          generateUserFullName(context, person?.name, person?.displayName, fetchInstanceNameFromUrl(person?.actorId)),
+          maxLines: 1,
+          overflow: TextOverflow.ellipsis,
+        ),
+        contentPadding: EdgeInsets.zero,
+      ),
+    );
+  }
+}
+
+/// The actions of the app bar for the account page
+class AccountAppBarUserActions extends StatelessWidget {
+  const AccountAppBarUserActions({super.key, this.showSaved = false, this.onToggleSaved});
+
+  /// Whether to show saved posts/comments
+  final bool showSaved;
+
+  /// Callback when the saved icon is tapped
+  final void Function(bool showSaved)? onToggleSaved;
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = AppLocalizations.of(context)!;
+    final feedBloc = context.read<FeedBloc>();
+
+    return Row(
+      children: [
+        IconButton(
```

Comment:
Any chance you could add a tooltip here?

Reply:
Done in https://github.com/thunder-app/thunder/pull/1597/commits/a736b9d0eff5dcfff5a689c9d993091c88fb2076!